README.rst

```diff
@@ -130,7 +130,20 @@ These effect how we display the Table of Contents in the side bar. You can read
 * ``includehidden`` Bool. Specifies if the sidebar includes toctrees marked with the `:hidden:` option
 * ``titles_only`` Bool. If True, removes headers within a page from the sidebar.
 
-Page-level configuration
+HTML Context Configuration
+--------------------------
+
+.. code:: python
+
+    html_context = {
```

Comment:
Is it technically possible to move this option (and the other VCS stuff later) to `html_theme_options` instead of `html_context`? It can be confusing to users that there are two places where project-wide configuration changes can be made.

Reply:
I think so the way I see it is `html_context` are things that we would want to know about the project while `html_theme_options` are things we want to configure how the theme looks and interacts. So I am ok moving this new option the `html_theme_options` but I think the rest should stay.